examples/python/helloworld/greeter_server.py

```diff
@@ -28,10 +28,12 @@ def SayHello(self, request, context):
 
 
 def serve():
+    port = '50051'
     server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
     helloworld_pb2_grpc.add_GreeterServicer_to_server(Greeter(), server)
-    server.add_insecure_port('[::]:50051')
+    server.add_insecure_port('[::]:' + port)
     server.start()
+    print("Server started, listening on " + port)
```

Comment:
If we're going to be reporting the port that we're listening on, we want the gRPC library to tell us the resolved port. In the case of port 50051, this will also be 50051. But If `port` is 0, the resolved port may be anything: 

`resolved_port = server.add_insecure_port('[::]:' + port)`

Then print `resolved_port`. You may also consider letting port be specified as a flag for the process.

Reply:
Make sense, I think fixed port is enough for quick start purpose, what do you think?